contribute/developer-guide.md

```diff
@@ -76,7 +76,10 @@ When you log in for the first time, Grafana asks you to change your password.
 
 #### Building on Windows
 
-The Grafana backend includes Sqlite3 which requires GCC to compile. So in order to compile Grafana on Windows you need to install GCC. We recommend [TDM-GCC](http://tdm-gcc.tdragon.net/download).
+The Grafana backend includes SQLite which requires GCC to compile. So in order to compile Grafana on Windows you need to install GCC. We recommend [TDM-GCC](http://tdm-gcc.tdragon.net/download). Eventually, if you use [Scoop](https://scoop.sh), you can install GCC through that.
+
+In order to use the `make` command above, you need to install [Make for Windows](http://gnuwin32.sourceforge.net/packages/make.htm) and use it in a Unix shell (f.ex. Git Bash). 
```

Comment:
In order to use the `make` command, install [Make for Windows](http://gnuwin32.sourceforge.net/packages/make.htm) and use it in a Unix shell (f.ex. Git Bash).

Reply:
FYI, I got a suspicious link warning when I clicked the link for Make for Windows.

Also, that program is from 2006. Do we really want to be recommending this to people?

Reply:
@oddlittlebird To my knowledge, that's the only alternative for using Make on Windows.

Reply:
@achatterjee-grafana I think my original wording is preferred because we don't want to instruct people to use Make, but inform them of _how_ they can do it if they wish. It's an alternative, not a requirement.

Reply:
It would seem the link sends the browser to 'http' instead of 'https' which doesn't resolve.  It would seem to be a browser certificate warning.

Reply:
Interesting that this page still doesn't have HTTPS. Will that have any security implications though for the intended use, i.e. to download Make? @ChrisDGH

Reply:
When clicking on the Make for Windows, it goes to the http domain.  In there, when clicking on the dependency zip files, it forward to https://sourceforge.net/projects/gnuwin32/files/make/3.81/make-3.81-dep.zip/download which is https.  From a security perspective, the http may warn in the browser, but the links forward to https.

Reply:
> @achatterjee-grafana I think my original wording is preferred because we don't want to instruct people to use Make, but inform them of _how_ they can do it if they wish. It's an alternative, not a requirement. Your suggestion looks more instructive to me, and thus more confusing than the current wording.
> 
> For example, I personally don't use Make on Windows to work with Grafana, but others (f.ex. @AgnesToulet) do.

@aknuds1, although I do not agree with you,  I will let this go. Product documentation is supposed to be instructive :)

Reply:
This is not product documentation though, it's a guide for developers. Development isn't a straightforward procedure, there are many ways of arriving at a result.

Reply:
@aknuds1 , if you install a Linux subsystem on your Windows machine, then you can install and run make. I use the Ubuntu subsystem and have been able to run some make commands in the past. (That said, they also have occasionally failed due to permissions problems, but that could be because I'm a Linux noob.)

Reply:
I haven't used the Linux subsystem so much on Windows, but it's essentially a VM isn't it, so you're actually on Linux? When you're doing that, you're not technically doing Windows development, so the Windows instructions don't apply as far as I understand.